src/integration/kotlin/pl/edu/agh/gem/integration/controller/AuthControllerIT.kt

```diff
@@ -263,7 +267,70 @@ class AuthControllerIT(
         // then
         response shouldHaveHttpStatus BAD_REQUEST
         response shouldHaveErrors {
-            errors[0].code shouldBe VerificationException::class.simpleName
+            errors.first().code shouldBe VerificationException::class.simpleName
+        }
+    }
+    should("send verification email") {
+        // given
+        val email = "email@email.com"
+        saveNotVerifiedUser(
+            email = email,
+            updatedCodeAt = LocalDateTime.now().minusMinutes(10),
```

Comment:
could we use more determisnist cases in test, instead of .now() use consistent time with .of()

Reply:
but we compare value of updatedCodeAt with LocalDateTime.now() in order to  check if we can send another mail, so i think we can't use deterministic value of date.

Reply:
https://kotest.io/docs/extensions/test_clock.html